tests/integration/test.replication.js

```diff
@@ -729,6 +729,155 @@ adapters.forEach(function (adapters) {
       }).catch(done);
     });
 
+    it('Test replication resumes when checkpointing is enabled', function (done) {
+      var db = new PouchDB(dbs.name);
+      var remote = new PouchDB(dbs.remote);
+
+      var expectedSince = false;
+      interceptChanges(db, function (opts) {
+        if (expectedSince !== false) {
+          opts.since.should.equal(expectedSince);
+          expectedSince = false;
+        }
+      });
+
+      db.bulkDocs({ docs: docs.slice(0, 1) })
+        .then(function () {
+          PouchDB.replicate(db, remote)
+            .on('error', done)
+            .on('complete', function (result) {
```

Comment:
Does the result contain anything that could be useful, like `seq_written` or something?